### PR TITLE
end to end test of interactive commands (.apply and .delete)

### PR DESCRIPTION
**Pull request by apesternikov**

### CODE REVIEW

File: .github/workflows/ci.yaml [hunk 1]
```diff
@@ -37,7 +37,12 @@ jobs:
           chmod +x "${TOOLS_DIR}/bin/bazel"
           curl -Ls -o "${TOOLS_DIR}/bin/buildifier" "https://github.com/bazelbuild/buildtools/releases/download/0.29.0/buildifier"
           chmod +x "${TOOLS_DIR}/bin/buildifier"
+          curl -Ls -o "${TOOLS_DIR}/bin/kind" "https://github.com/kubernetes-sigs/kind/releases/download/v0.7.0/kind-linux-amd64"
+          chmod +x "${TOOLS_DIR}/bin/kind"
           echo "::add-path::${TOOLS_DIR}/bin"
+      - name: Setup kind cluster
+        run: |
+          ./create_kind_cluster.sh
```
Comment (kzadorozhny) [inline .github/workflows/ci.yaml:45]: I'd move this stage right before E2E test or within E2E stage

File: e2e_test.sh [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+go get sigs.k8s.io/kind@v0.7.0
+
+cluster_running="$(docker inspect -f '{{.State.Running}}' kind-control-plane 2>/dev/null || true)"
+if [ "${cluster_running}" != 'true' ]; then
+  ./create_kind.cluster.sh
+fi
+
+delete() {
+    echo "Deleting kind cluster"
+    # kind delete cluster
```
Comment (kzadorozhny) [inline e2e_test.sh:35]: Does it meant to be left commented out?

File: examples/.gitignore [hunk 3]
```diff
@@ -0,0 +1 @@
+cloud
```
Comment (kzadorozhny) [inline examples/.gitignore:1]: should be `cloud/`

File: .github/workflows/ci.yaml (hunk 1 above)
Comment (apesternikov) [reply .github/workflows/ci.yaml:45]: The k8s cluster could be used for regular IT testing, not just E2E

**Pull request merged**